rel-sign.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
+set -eu
+
+# Make temporary folder
+_mktemp() {
+    local base=${1:-/tmp}
+    if [[ $(uname) = Darwin ]]; then mktemp -d $base/rel-sign.XXXXXXXXXX
+    else TMPDIR="$base" mktemp -d -t rel-sign.XXXXXXXXXX
+    fi
+}
+
+# Print help message
+function print_help() {
+    printf \
+"Sign releases from the GitHub repository with PGP key.
+Usage: 
+    ./rel-sign.sh -h, --help
+    ./rel-sign.sh -r=user/repository -v=0.1.1 -k=7761E36F86C935A6
+    -r, --repo - repository, in the format username/repository
+    -v, --version - version, in the format x.y.z, without leading v
+    -k, --key - signing key id, fingerprint or email
+    -gpg - use gpg instead of rnp for signing.
+    -s, --src - use specified folder for release sources comparison instead of downloading from GitHub.
+    -pparams - addition OpenPGP params, like '--homedir .rnp', '--keyfile seckey.asc', etc.\n"
+}
+
+# Extract parameters
+if [ $# -eq 0 ]; then
+    print_help
+    exit 0
+fi
+
+PPARAMS=""
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    --repo*|-r*)
+      if [[ "$1" != *=* ]]; then shift; fi # Value is next arg if no `=`
+      REPO="${1#*=}"
+      REPOLAST=`basename ${REPO}`
+      ;;
+    --version*|-v*)
+      if [[ "$1" != *=* ]]; then shift; fi
+      VERSION="${1#*=}"
+      ;;
+    --key*|-k*)
+      if [[ "$1" != *=* ]]; then shift; fi
+      KEY="${1#*=}"
+      ;;
+    --gpg)
+      USEGPG=1
+      ;;
+    --src*|-s*)
+      if [[ "$1" != *=* ]]; then shift; fi
+      SRCDIR=`realpath "${1#*=}"`
+      if [ ! -d ${SRCDIR} ]; then
+          printf "Directory ${SRCDIR} doesn't exist.\n"
+          exit 1
+      fi
+      printf "Comparing with sources from ${SRCDIR}\n"
+      ;;
+    --pparams*)
+      if [[ "$1" != *=* ]]; then shift; fi
+      PPARAMS="${1#*=}"
```

Comment:
ditto for `$PPARAMS`

Reply:
If considering to use array for `PPARAMS`, this might become `PPARAMS=("$@")` instead.

```bash
      if [[ "$1" != *=* ]]; then
        shift
        PPARAMS=("$@")
      else
        PPARAMS=("${1#*=}")
      fi
```

Reply:
Thanks, agree, let's better use array for PPARAMS.